tests/code/test_minor.py

```diff
@@ -132,26 +134,345 @@ def test_getCommunityEngagementByTerm():
+                                      "selectedSummerTerm": "Summer 2021",
+                                     }
+        
+        saveSummerExperience('partont', newPartontSummerExperience, 'ramsayb2')
+
+        getNewPartontSummerExperience = IndividualRequirement.select()
+        assert getNewPartontSummerExperience.count() == 1
+        assert getNewPartontSummerExperience[0].username_id == 'partont'
+        assert getNewPartontSummerExperience[0].description == 'Second Summer Experience for Tyler'
+
+        # Add a summer experience for another studnet and verify both students have summer experience records
+        
+        neillzSummerExperience = {"summerExperience": "Summer Experience for Zach",
+                                  "selectedSummerTerm": "Summer 2021"
+                                  }
+        saveSummerExperience('neillz', neillzSummerExperience, 'ramsayb2')
+        getNeillzSummerExperience = IndividualRequirement.select()
+        assert getNeillzSummerExperience.count() == 2
+        assert getNeillzSummerExperience[0].username_id == 'partont'
+        assert getNeillzSummerExperience[1].username_id == 'neillz'
+        assert getNeillzSummerExperience[1].description == "Summer Experience for Zach"
+
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_getSummerExperience():
+    with mainDB.atomic() as transaction:
+        IndividualRequirement.delete().execute()
+
+        partontSummerEngagement = {"username": "partont",
+                                   "program": None,
+                                   "course": None, 
+                                   "description": "Summer engagement",
+                                   "term": 3,
+                                   "requirement": 16,
+                                   "addedBy": "ramsayb2",
+                                   "addedOn": ""
+                                  }
+        IndividualRequirement.create(**partontSummerEngagement)
+        tylerSummerEngagement = getSummerExperience('partont')
+
+        assert tylerSummerEngagement[1] == "Summer engagement"
+        
+        IndividualRequirement.update(description = "Updated summer engagement").where(IndividualRequirement.username == 'partont').execute()
+        tylerUpdatedSummerEngagement = getSummerExperience('partont')
+
+        assert tylerUpdatedSummerEngagement[1] == "Updated summer engagement"
+        
+        IndividualRequirement.update(term = 6).where(IndividualRequirement.username == 'partont').execute()
+        tylerUpdatedSummerEngagementTerm = getSummerExperience('partont')
+
+        assert tylerUpdatedSummerEngagementTerm[0] == 'Summer 2022'
+
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_removeSummerExperience():
+    with mainDB.atomic() as transaction:
+        # remove the summer experience for khatts that is in test data
+
+        removeSummerExperience('khatts')
```

Comment:
Again, maybe not the biggest fan of just assuming what we have in test data is what we want? Feels like it could be better if we manually give someone a summer experience and then remove it. Or we can make sure before removing that it is there I think this would be a little bit better.